shell/platform/darwin/ios/framework/Source/FlutterResourcePlugin.mm

```diff
@@ -0,0 +1,95 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/darwin/ios/framework/Source/FlutterResourcePlugin.h"
+#include "flutter/fml/logging.h"
+
+#include <Foundation/Foundation.h>
+#include <UIKit/UIKit.h>
+
+@implementation FlutterResourcePlugin
+
+- (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
+  NSString* method = call.method;
+  id args = call.arguments;
+  if ([method isEqualToString:@"SystemImage.load"]) {
```

Comment:
Please pull this string literal into a variable so that the test can reference it.